src/sentry/static/sentry/app/views/settings/components/forms/model.tsx

```diff
@@ -219,7 +220,7 @@ class FormModel {
       return '';
     }
 
-    return this.fields.get(id);
+    return this.fields.get(id) as FieldValue;
```

Comment:
Is there a template parameter we can give `this.fields` so that it knows it returns a `FieldValue` instead of doing this